lib/InputWithButton/index.js

```diff
@@ -23,13 +23,18 @@ class InputWithButton extends Component {
       buttonTextAfterClick,
       inputId,
       buttonId,
-      disabled
+      disabled,
+      paddingSmall
     } = this.props;
 
     const { hasBeenClicked } = this.state;
 
     return (
-      <div className="input-with-button">
+      <div
+        className={cx('input-with-button', {
```

Comment:
Is it worth assigning this in a `classNames` const?

Reply:
Since its just one condition I dont really see a problem with it, but happy to change it